steering/elections/2022/election_desc.md

```diff
@@ -0,0 +1,34 @@
+# 2022 Istio Steering Committee election
```

Comment:
this may be a lot of text for the election page.  Go ahead and merge it, but you might need to abbreviate it and put the rest on a page in the community repo.  Take a look at the election page display once it's merged.

Reply:
It's already just copied and pasted from the election page; I'd have to see how it renders to judge